provisioning/service/src/Config/BulkOperationMode.cs

```diff
@@ -2,22 +2,25 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System.Runtime.Serialization;
 
 namespace Microsoft.Azure.Devices.Provisioning.Service
 {
     /// <summary>
     /// The Device Provisioning Service bulk operation modes.
     /// </summary>
     /// <seealso cref="https://docs.microsoft.com/en-us/rest/api/iot-dps/deviceenrollment">Device Enrollment</seealso>
+    [JsonConverter(typeof(StringEnumConverter))]
     public enum BulkOperationMode
     {
-        [JsonProperty(PropertyName = "create")]
-        Create,
-        [JsonProperty(PropertyName = "update")]
+        [EnumMember(Value = "create")]
+        Create = 1,
```

Comment:
Why is create = 1? Is the int value of any importance for serialization?

Reply:
No it is not necessary

Reply:
Removed